src/js/Templates/Shared/Image.js

```diff
@@ -80,7 +85,9 @@ export default class Image extends React.Component {
                     </div>
                 )
             } else {
-                return <img className={this.props.class} src={this.props.image} />
+                return (
+                    <img className={this.props.class} src={this.props.image} onError={e => this.onError(e)} />
```

Comment:
I think the onError handler needs to be added to the template image tag as well.